operations/network/management_network/apply_custom_config_1.2.md

```diff
@@ -41,27 +42,27 @@ interface 1/1/36
 
 If the switch had `system interface-group` commands those would be added here.
 
-```
+```console
 sw-spine-001(config)# system interface-group 3 speed 10g
 ```
 
 `vrf Customer` will be appended to the default route configuration.
 
-```
+```console
 sw-spine-001# conf t
 sw-spine-001(config)# ip route 0.0.0.0/0 10.101.15.141 vrf Customer
 ```
 
-```
+```console
 sw-spine-002# conf t
 sw-spine-002(config)# ip route 0.0.0.0/0 10.101.15.189 vrf Customer
 ```
 
 ## Mellanox
```

Comment:
The later Aruba/Mellanox/etc sections have headings which explain what is being done in those sections. The same should be done here.